seed.js

```diff
@@ -1,15 +1,36 @@
 // This file allows us to seed our application with data
 // simply run: `node seed.js` from the root of this project folder.
+//app will talk to server which will talk to seed.js
+var db = require('./models');
+var personalData = [];
 
-// var db = require('./models');
+personalData.push({name: 'David Jue'});
```

Comment:
Again, rather than use seeded DB data, this should have been a hardcoded object.